packages/config/config/devices/0x0330/zv9001k4-dim-g2.json

```diff
@@ -0,0 +1,37 @@
+// Sunricher ZV9001K4-DIM-G2
+// 2 group single color wall mounted remote
+{
+	"manufacturer": "Sunricher",
+	"manufacturerId": "0x0330",
+	"label": "ZV9001K4-DIM-G2",
+	"description": "2 group single color wall mounted remote",
+	"devices": [
+		{
+			"productType": "0x0003",
+			"productId": "0xa306",
+			"zwaveAllianceId": 3783
+		}
+	],
+	"firmwareVersion": {
+		"min": "0.0",
+		"max": "255.255"
+	},
+	"associations": {
+		"1": {
+			"label": "Lifeline",
+			"maxNodes": 1,
+			"isLifeline": true
+		},
+		"2": {
+			"label": "Launch 1",
+			"maxNodes": 5
+		},
+		"3": {
+			"label": "Launch 2",
+			"maxNodes": 5
+		}
+	},
```

Comment:
These devices support Association Group Info CC and Z-Wave+, so associations can be auto-discovered. Unless the auto-discovered labels are worse than this, the entire block here is unnecessary.